Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -22,6 +23,18 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         [Tooltip("Supply a friendly name for the anchor as the key name for the WorldAnchorStore.")]
         public string SavedAnchorFriendlyName = "SavedAnchorFriendlyName";
 
+        [Tooltip("Place parent on tap instead of current game object.")]
+        public bool PlaceParentOnTap = false;
+
+        [Tooltip("Specify the parent game object to be moved on tap, if the immediate parent is not desired.")]
+        public GameObject ParentGameObjectToPlace;
+
+        /// <summary>
+        /// Keeps track of if the user is moving the object or not.
+        /// Setting this true will place the object immediately.
```

Comment:
I was just concerned with word "enable" getting confused with "allow" so that the comment will read something like 

> Setting this to true will allow the user to place the object

instead of 

> Setting this to true will place the object immediately